cmd/terramate/cli/run.go

```diff
@@ -373,7 +373,7 @@ func (c *cli) syncLogs(logger *zerolog.Logger, run runContext, logs cloud.Deploy
 	logger.Debug().RawJSON("logs", data).Msg("synchronizing logs")
 	ctx, cancel := context.WithTimeout(context.Background(), defaultCloudTimeout)
 	defer cancel()
-	stackID := c.cloud.run.meta2id[run.Stack.ID]
+	stackID := c.cloud.run.meta2id[strings.ToLower(run.Stack.ID)]
```

Comment:
Could we a test for testing this bugfix? Syncing a deployment for a stack whose ID has upper case and their logs?

Reply:
we tested this on staging and it works as expected.

Reply:
In my humble opinion, we should add this in long term to interop tests

Reply:
I agree, in the interest of time I will move forward with this PR now and add interop tests later.